mage_ai/frontend/components/ActionForm/index.tsx

```diff
@@ -199,7 +200,7 @@ function ActionForm({
                 minLines={MIN_LINES_ACTIONS}
                 mode="python"
                 onChange={e => {
-                  payload['action_code'] = e;
+                  payload['action_code'] = e.replace('\t', ' '.repeat(4));
```

Comment:
Concerns about this approach:
- Are there cases where a tab is actually preferred in Python source code over 4 spaces? I wouldn't think so, since code editors anyways insert 4 spaces in place of a tab.
- Is there any performance hit for doing this in `onChange`? Don't think so either.

Reply:
Seems fine